package.json

```diff
@@ -31,6 +31,7 @@
     "@google-cloud/resource-manager": "^4.1.1",
     "@google-cloud/storage": "^6.7.0",
     "chai": "^4.3.7",
+    "commander": "^10.0.0",
```

Comment:
Since this is a GitHub Action, why don't you try out:

https://nodejs.org/docs/latest-v18.x/api/util.html#utilparseargsconfig

Rather than pulling in a dep, you just need to set the Node version to 18.

Reply:
The workflow generation process actually happens offline in the dev environment. So, I'm not sure this will work. The user might be running a lower version of Node.